app/uk/gov/hmrc/exports/models/MovementNotification.scala

```diff
@@ -18,15 +18,18 @@ package uk.gov.hmrc.exports.models
 
 import org.joda.time.DateTime
 import play.api.libs.json.Json
+import reactivemongo.bson.BSONObjectID
+import uk.gov.hmrc.mongo.json.ReactiveMongoFormats
+import uk.gov.hmrc.mongo.json.ReactiveMongoFormats.mongoEntity
 import uk.gov.hmrc.wco.dec.{EntryStatus, GoodsItem, InventoryLinkingMovementResponse, UcrBlock}
 
 case class MovementNotification(
-  dateTimeReceived: DateTime = DateTime.now(),
-  conversationId: String,
-  eori: String,
-  badgeId: Option[String] = None,
-  movementResponse: InventoryLinkingMovementResponse
-)
+                                 dateTimeReceived: DateTime = DateTime.now(),
```

Comment:
Format in whole file, previous one is better, please always use autoformatter -> `Option` + `Command` + `L`